mos-platform/atari5200-super/clang.cfg

```diff
@@ -0,0 +1,8 @@
+#
+# Allow for 96 usable bytes in the zp, using the same layout and
+# offset as Atari 800 for now to make debugging and analysis easier.
+#
+# Strictly speaking as the lowest 24 are used, we have 256-32-24
+# leaving 200 free.
+-mlto-zp=96
```

Comment:
Do we want to keep this?

Reply:
I do for now, but not long-term.

I use the "sameness" to the other platform when looking at hexdumps and disassembly, something mostly familiar.